lib/components/explorer_result_page_qubic_id/explorer_result_page_qubic_id.dart

```diff
@@ -15,19 +15,19 @@ class ExplorerResultPageQubicId extends StatelessWidget {
   });
   final DateFormat formatter = DateFormat('dd MMM yyyy \'at\' HH:mm:ss');
   final ExplorerIdInfoDto idInfo;
-
+  //TODO Show latest transfers for a Public ID here
   Widget listTransactions() {
     return Column(
       mainAxisSize: MainAxisSize.max,
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        for (var transaction in idInfo.latestTransfers!)
-          Padding(
-            padding: const EdgeInsets.only(bottom: ThemePaddings.normalPadding),
-            child: ExplorerResultPageTransactionItem(
-                transaction: transaction, isFocused: false, showTick: true),
-          ),
-        const SizedBox(height: ThemePaddings.normalPadding)
+        // for (var transaction in idInfo.latestTransfers!)
```

Comment:
do we need this commented?

Reply:
Done, removed the commented code keeping just the TODO comment